aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -501,6 +501,97 @@ public void testColumnCommentsAndParameters() {
     assertThat(actualColumns).isEqualTo(expectedColumns);
   }
 
+  @Test
+  public void testGlueTableColumnCommentsPreserved() {
```

Comment:
What about the case where we drop a column from a table. I believe today we keep those columns and descriptions. Should we add a test for that?